plugins/pdf_reader/plugin.spec.yaml

```diff
@@ -29,6 +29,7 @@ actions:
Ye6xMmfwNTJA+qENa7h/Tlxyv7inkuzDtOx4d+Py4xrAlcH1MC5850FxNy4/6Ve2iyNfJGsedT3qffSeRx961JR9oPjAmQfE4skzJ4UXp89OC/lQgzuT9rrT3Z9y17bWDFtaxWEzukHvSu9o3eZgdERxj6DQ7Yeb3Ye7G9zrWsuHTZiwhIJ20S22iwNiRnxYPCtarAdCG9yDuM6HLoUEJVRSGrQPuAf8A+L80nkl0edBa3uze4t7xd5gg7une7vb3u3u9nd/p/un3Re7zSPd5Bn8Bl8Mng2KSrDBH1SCGzzBG3tcw1WtlcNOYh92tNqHBYKNboVhv33JLtjtI/b77KId2kEoVhETmSdnZg8Oeb1985alA32aNXS7Rk5pdUPsrgwe1synNBg+fHt4lpA/jZw8fRo61vdpLUNhLbo+0qfFEVAYUETAsX62Cjoi+XzByy/i9SI8hXfwTnmReDSvU2GFD948yeMRledKxMsEdJzg3ct4SGB6BLWP5oHdGNOrKzHtvGGOK+s3DtQc/W+sYVh9CmVuZHN0cmVhbQplbmRvYmoKCjYgMCBvYmoKNDkwMgplbmRvYmoKCjcgMCBvYmoKPDwvVHlwZS9Gb250RGVzY3JpcHRvci9Gb250TmFtZS9CQUFBQUErTGliZXJhdGlvblNlcmlmCi9GbGFncyA0Ci9Gb250QkJveFstNTQzIC0zMDMgMTI3NyA5ODFdL0l0YWxpY0FuZ2xlIDAKL0FzY2VudCA4OTEKL0Rlc2NlbnQgLTIxNgovQ2FwSGVpZ2h0IDk4MQovU3RlbVYgODAKL0ZvbnRGaWxlMiA1IDAgUgo+PgplbmRvYmoKCjggMCBvYmoKPDwvTGVuZ3RoIDI1MS9GaWx0ZXIvRmxhdGVEZWNvZGU+PgpzdHJlYW0KeJxdkMlqwzAQhu96Ch3TQ5DsJA4FYygpAR+6ULcPIEtjV1BLYiwf/PYdyWkLPUh8s/yziUv72DobxSt63UHkg3UGYfYLauA9jNaxouTG6niz8q8nFZggbbfOEabWDb6umXij2Bxx5bsH43u4Y+IFDaB1I999XDqyuyWEL5jARS5Z03ADA9V5UuFZTSCyat8aCtu47knyl/C+BuBltottFO0NzEFpQOVGYLWUDa+v14aBM/9i1aboB/2pkDILypTyVDbEZeaqSHzIfJaJj5v/PvFp42PiKvPhnPvcKqaO6SQ/m3C9INIW+W55/DS4dfB72uBDUuX3Dbs0ePoKZW5kc3RyZWFtCmVuZG9iagoKOSAwIG9iago8PC9UeXBlL0ZvbnQvU3VidHlwZS9UcnVlVHlwZS9CYXNlRm9udC9CQUFBQUErTGliZXJhdGlvblNlcmlmCi9GaXJzdENoYXIgMAovTGFzdENoYXIgNgovV2lkdGhzWzc3NyA2NjYgNDQzIDUwMCAyNzcgNTAwIDUwMCBdCi9Gb250RGVzY3JpcHRvciA3IDAgUgovVG9Vbmljb2RlIDggMCBSCj4+CmVuZG9iagoKMTAgMCBvYmoKPDwvRjEgOSAwIFIKPj4KZW5kb2JqCgoxMSAwIG9iago8PC9Gb250IDEwIDAgUgovUHJvY1NldFsvUERGL1RleHRdCj4+CmVuZG9iagoKMSAwIG9iago8PC9UeXBlL1BhZ2UvUGFyZW50IDQgMCBSL1Jlc291cmNlcyAxMSAwIFIvTWVkaWFCb3hbMCAwIDU5NS4zMDM5MzcwMDc4NzQgODQxLjg4OTc2Mzc3OTUyOF0vR3JvdXA8PC9TL1RyYW5zcGFyZW5jeS9DUy9EZXZpY2VSR0IvSSB0cnVlPj4vQ29udGVudHMgMiAwIFI+PgplbmRvYmoKCjQgMCBvYmoKPDwvVHlwZS9QYWdlcwovUmVzb3VyY2VzIDExIDAgUgovTWVkaWFCb3hbIDAgMCA1OTUgODQxIF0KL0tpZHNbIDEgMCBSIF0KL0NvdW50IDE+PgplbmRvYmoKCjEyIDAgb2JqCjw8L1R5cGUvQ2F0YWxvZy9QYWdlcyA0IDAgUgovT3BlbkFjdGlvblsxIDAgUiAvWFlaIG51bGwgbnVsbCAwXQovTGFuZyhwbC1QTCkKPj4KZW5kb2JqCgoxMyAwIG9iago8PC9DcmVhdG9yPEZFRkYwMDU3MDA3MjAwNjkwMDc0MDA2NTAwNzI+Ci9Qcm9kdWNlcjxGRUZGMDA0QzAwNjkwMDYyMDA3MjAwNjUwMDRGMDA2NjAwNjYwMDY5MDA2MzAwNjUwMDIwMDAzNjAwMkUwMDM0PgovQ3JlYXRpb25EYXRlKEQ6MjAyMTExMjgyMjQ5NDIrMDEnMDAnKT4+CmVuZG9iagoKeHJlZgowIDE0CjAwMDAwMDAwMDAgNjU1MzUgZiAKMDAwMDAwNjAwOSAwMDAwMCBuIAowMDAwMDAwMDE5IDAwMDAwIG4gCjAwMDAwMDAxOTcgMDAwMDAgbiAKMDAwMDAwNjE3OCAwMDAwMCBuIAowMDAwMDAwMjE3IDAwMDAwIG4gCjAwMDAwMDUyMDMgMDAwMDAgbiAKMDAwMDAwNTIyNCAwMDAwMCBuIAowMDAwMDA1NDE5IDAwMDAwIG4gCjAwMDAwMDU3MzkgMDAwMDAgbiAKMDAwMDAwNTkyMiAwMDAwMCBuIAowMDAwMDA1OTU0IDAwMDAwIG4gCjAwMDAwMDYyNzcgMDAwMDAgbiAKMDAwMDAwNjM3NCAwMDAwMCBuIAp0cmFpbGVyCjw8L1NpemUgMTQvUm9vdCAxMiAwIFIKL0luZm8gMTMgMCBSCi9JRCBbIDw2RDVGRjZBMERFNDcxNUEyRUZCOURGN0E3MjNFRUFFQT4KPDZENUZGNkEwREU0NzE1QTJFRkI5REY3QTcyM0VFQUVBPiBdCi9Eb2NDaGVja3N1bSAvNDkzMUY3MDAzRjc4RDhBMERENzk2RjlDMTE5RTlDOEQKPj4Kc3RhcnR4cmVmCjY1NDkKJSVFT0YK
```

Comment:
Could use a shorter example if this appears in any UIs
Something like `this is a 9376 length value`

Reply:
@cmcnally-r7 I can't shorten it any more as it's the encoded whole PDF, not just its contents.